### PR TITLE
[docs]: add studio desktop callout

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/docs/cloud/quick_start.md [hunk 1]
```diff
@@ -66,6 +66,12 @@ Now that we have set everything up on our local file system, we are ready to hos
 
 ## Test the graph build locally
 
+### Using LangGraph Studio Desktop (recommended)
+
+Testing your graph locally is incredibly easy to do by using LangGraph Studio Desktop. The studio provides an easy-to-use user interface and allows quick development cycles for auto updating your graph when you modify code, which makes it easier to iterate and test your graph. In addition, the studio provides in depth trace views of each graph run, which makes it easy to debug your graph by inspecting individual steps. You can download the app [here](https://langgraph-studio.vercel.app/api/mac/arm64/latest) and can read about how to use it in [this walkthrough](https://github.com/langchain-ai/langgraph-studio/blob/main/README.md).
```
Comment (vbarda) [inline docs/docs/cloud/quick_start.md:71]: i would maybe just copy-paste this wording from studio

"LangGraph Studio offers a new way to develop LLM applications by providing a specialized agent IDE that enables visualization, interaction, and debugging of complex agentic applications

With visual graphs and the ability to edit state, you can better understand agent workflows and iterate faster. LangGraph Studio integrates with [LangSmith](https://smith.langchain.com/) so you can collaborate with teammates to debug failure modes."

Comment (vbarda) [reply docs/docs/cloud/quick_start.md:71]: also trace views might be confused with langsmith traces so let's not say that

Comment (isahers1) [reply docs/docs/cloud/quick_start.md]: yup, just copy pasted and added a screenshot

File: docs/docs/cloud/index.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ The LangGraph Cloud API exposes functionality of your LangGraph application thro
 
 LangGraph Cloud is seamlessly integrated with [LangSmith](https://www.langchain.com/langsmith) and is accessible from within the LangSmith UI.
 
+LangGraph Cloud projects can be tested and debugged using the [LangGraph Studio Desktop](https://github.com/langchain-ai/langgraph-studio).
```
Comment (vbarda) [inline docs/docs/cloud/index.md:26]:
```suggestion
LangGraph Cloud applications can be tested and debugged using the [LangGraph Studio Desktop](https://github.com/langchain-ai/langgraph-studio). 
```

**Review by vbarda — APPROVED** [review]

**Pull request merged**